src/metadata/entity/index.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  Column,
+  Entity,
+  JoinTable,
+  ManyToMany,
+  PrimaryColumn,
+} from 'typeorm'
+
+@Entity()
+export class IasqlDatabase {
+  // the name we assign
+  @PrimaryColumn()
+  pgName: string;
+
+  // the alias the user assigns
+  @Column()
+  alias: string;
+
+  @Column()
+  region: string;
+
+  @Column()
+  pgUser: string
+}
+
+@Entity()
+export class IasqlUser {
+  // auth0 generated id
+  @PrimaryColumn()
+  id: string;
+
+  @Column()
+  email: string;
+
+  @ManyToMany(() => IasqlDatabase, {
```

Comment:
made this many-to-many from the get-go for when we eventually want to have multiple users access the same database, but technically every database is associated with a single user atm